src/program/services/downloaders/alldebrid.py

```
@@ -532,106 +712,133 @@ def get_torrent_info(self, torrent_id: int) -> TorrentInfo:
-                    self.filename = filename
-                    self.filesize = filesize
-
             return UnrestrictedLink(
                 download=unrestricted_url,
-                filename=getattr(link_data, "filename", "file"),
-                filesize=getattr(link_data, "filesize", 0),
+                filename=link_data.filename,
+                filesize=link_data.filesize,
             )
 
         except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)